bpf/bpf_lxc.c

```diff
@@ -1053,11 +1053,13 @@ static __always_inline int handle_ipv4_from_lxc(struct __ctx_buff *ctx, __u32 *d
 #if defined(ENABLE_HOST_FIREWALL) && !defined(ENABLE_ROUTING)
 	/* If the destination is the local host and per-endpoint routes are
 	 * enabled, jump to the bpf_host program to enforce ingress host policies.
+	 * Note that bpf_lxc can be loaded before bpf_host, so bpf_host's policy
+	 * program may not yet be present at this time.
 	 */
 	if (*dst_sec_identity == HOST_ID) {
 		ctx_store_meta(ctx, CB_FROM_HOST, 0);
 		tail_call_static(ctx, &POLICY_CALL_MAP, HOST_EP_ID);
-		return DROP_MISSED_TAIL_CALL;
+		return DROP_HOST_NOT_READY;
```

Comment:
Will this case, and the `send_drop_notify` case below always be as a result of the host policy program not being in place?

Reply:
Yes, if calling into POLICY_CALL_MAP at slot HOST_EP_ID fails, the host program has not been loaded.